frontends/verific/verific.cc

```diff
@@ -2901,6 +2914,17 @@ std::set<std::string> import_tops(const char* work, std::map<std::string,Netlist
 			}
 		}
 #endif
+
```

Comment:
This is the wrong place to put this. It only runs if "top" is not blank, it won't work in "auto-top" mode. You should move this to the `verific_import` function so that it runs for both `import_tops` and `import_all`

Reply:
Working on it